recipes/libsodium/libsodium.inc

```diff
@@ -0,0 +1,8 @@
+inherit autotools c pkgconfig
+
+DESCRIPTION = "libsodium - Sodium is a modern, easy-to-use software library for encryption, decryption, signatures, password hashing and more."
+HOMEPAGE = "https://download.libsodium.org/doc/"
+LICENSE = "ISC"
+DEPENDS_${PN} = "libc"
```

Comment:
This should be a '+=' assignment, which becomes important when library.oeclass is used, as it has a default DEPENDS_${PN} against ${PN}-dev_${PV}